README.rst

```diff
@@ -2,11 +2,141 @@
+If there is are new topics, ``community_mailbot`` will send an email the appropriate email lists using its buddy, `Mandrill <http://mandrillapp.com>`_.
+Before the bot goes back to sleep, it keeps note of all the topics it's emailed already in a simple JSON cache file.
+
+Installation
+------------
+
+Create a virtual environment running Python 3.5, then:
+
+.. bash::
+
+   pip install git+https://github.com/lsst-sqre/community_mailbot.git@tickets/DM-3690
+
+Running the community mailbot
+-----------------------------
+
+Keys to the Kingdom
+~~~~~~~~~~~~~~~~~~~
+
+You'll need to get API keys from community.lsst.org and Mandrill.
+Set them to the following environment variables:
+
+* ``$MANDRILL_KEY`` (note, use the API key for the ``community_mailbot`` subaccount)
+* ``$DISCOURSE_KEY`` (the Discourse key should correspond to a user with sufficient permissions)
+* ``$DISCOURSE_USER``
+
+Optionally set ``$COMMUNITY_MAILBOT_CACHE`` to the location where you want the Mailbot to keep track of its topics it has forwarded.
+
+Setup Topic → Email Mapping
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To setup how Discourse categories are mapped to recipient email addresses, you need to know the integer IDs for each relevant category.
+To get these, run the included script:
+
+.. code:: bash
+
+   discourse_categories
+
+If you set the aforementioned environment variables, you won't need to provide any arguments to the script.
+
+Next, create a ``config.json`` file.
+It's a simple hash structure; each key-value pair is the Discourse category ID and a list of email recipients for that category.
+The general format is:
+
+.. code:: text
+
+   {
+       "<id>": [{"email": "<email address>",
+                 "name": "<recipient name>",
+                 "type": "to"}],
+   }
+
+Note that since the recipient information for each category is a ``list``, you can have multiple recipients.
+
+Pre-cache old topics
+~~~~~~~~~~~~~~~~~~~~
+
+Before having the bot send emails, you'll want it to know about and ignore older messages.
+To warm up the cache, we'll run the ``forward_discourse`` script with the ``--cache-only``option.
+
+.. code:: bash
+
+   forward_discourse config.json --cache-only
+
+Note that ``forward_discourse`` can configure itself with the environment variables you've already setup.
+Run ``forward_discourse --help`` for the full set of options.
+
+Set up a cron schedule
+~~~~~~~~~~~~~~~~~~~~~~
+
+The bot works best when it regularly monitors a Discourse site for new topics.
+``cron`` is a great way to set this up.
+
+It's useful to create a shell script to contain all of the script arguments.
+For example, create a script called ``run_mailbot.sh``:
+
+.. code:: bash
+
+   #!/bin/bash
+   source /home/ec2-user/.bash_profile
+   source activate community_mailbot
+   forward_discourse /home/ec2-user/config.json
+
+   echo "$(date) Ran forward_discourse" > /home/ec2-user/forward_discourse.log
```

Comment:
Do you want ">>" here? Is this really necessary, since it's not really more informative than the cron log?